.jenkins/jenkins_buildbot_python2_debug.sh

```diff
@@ -10,7 +10,7 @@ export LD_LIBRARY_PATH=/usr/local/cuda/lib64:$LD_LIBRARY_PATH
 export LIBRARY_PATH=/usr/local/cuda/lib64:$LIBRARY_PATH
 
 GPUARRAY_CONFIG="Release"
-DEVICE=cuda0
```

Comment:
`DEVICE` is only read by libgpuarray / pygpu, which do not understand `"cuda"` anyway, they need the full name.
I think it was needed for running tests on the right device, but I do not think it is necessary for Theano's tests.
If using `cuda` worked, then I think we can delete the whole line.